pkg/apis/resource/types.go

```diff
@@ -1937,16 +1944,39 @@ type DeviceTaintSelector struct {
 	//
 	// +optional
 	Device *string
+}
 
-	// Selectors contains the same selection criteria as a ResourceClaim.
-	// Currently, CEL expressions are supported. All of these selectors
-	// must be satisfied.
+// DeviceTaintRuleStatus provides information about an on-going pod eviction.
+type DeviceTaintRuleStatus struct {
+	// Conditions provide information about the current state of the DeviceTaintRule
+	// in a machine-readable and human-readable format.
```

Comment:
Should we note the caveat here that this condition may lag behind reality? e.g. when a Pod is created that matches an existing DeviceTaintRule with EvictionInProgress=False, the DeviceTaintRule will be stale until the eviction controller catches up.

Reply:
Changed to:
```
	// Conditions provide information about the state of the DeviceTaintRule
	// and the cluster at some point in time,
	// in a machine-readable and human-readable format.
```

Reply:
Should we make the same change in staging/src/k8s.io/api/resource/v1alpha3/types.go?